src/actions/performApiDataRequest.test.ts

```diff
@@ -410,10 +434,14 @@ describe('performApiDataRequest', () => {
             jest.advanceTimersByTime(3000);
             return Promise.resolve(response1);
         });
-        await performApiRequest('getData', {}, { data: 'json' })(dispatch, () => state);
-        const error = new Error('Timeout');
-        const requestKey = getRequestKey('getData');
-        expect(dispatch).toHaveBeenCalledWith(apiDataFail(requestKey, error));
+        try {
+            await performApiRequest('getData', {}, {data: 'json'})(dispatch, () => state);
+        } catch {
+            const error = new Error('Timeout');
```

Comment:
soms zet je de expect in het catch gedeelte en soms erna, kan je dat consistent maken? Of heeft het een specifiek doel?